pecos/xmc/xtransformer/network.py

```diff
@@ -417,11 +424,80 @@ def forward(
         }
 
 
+@add_start_docstrings(
+    """DistilBert Model with mutli-label classification head on top for XMC.\n""",
+    DISTILBERT_START_DOCSTRING,
+)
+class DistilBertForXMC(BertPreTrainedModel):
```

Comment:
Parent class should be `DistilBertPreTrainedModel` rather than `BertPreTrainedModel`